src/main/java/nl/tudelft/pixelperfect/event/EventFactory.java

```diff
@@ -0,0 +1,61 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Created by woute on 5/10/2016.
+ */
+
+/**
+ * Not a factory as the design pattern, but used for id's.
+ * <p>
+ * Create a random event, with unique id's.
+ */
+public class EventFactory {
+
+  private int id;
+
+  /**
+   * Constructor for EventFactory.
+   */
+  public EventFactory() {
+    id = 0;
+  }
+
+  /**
+   * Generate a random event.
+   *
+   * @return an Event.
+   */
+  public Event randomEvent() {
+    int x = (int) Math.floor(Math.random() * 100) % 4;
+    return create(x);
+  }
+
+  /**
+   * Create an event based on the parameter.
+   *
+   * @param x Defines which event is to be created.
+   * @return an Event.
+   */
+  public Event create(int x) {
```

Comment:
completely agree with this, was never intended to permanent, but couldnt think of a nicer way for now